dev-docs/bidders/pubgears.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: Pubgears
+description: Pubgears Bidder Adapter
+pbjs: true
+biddercode: pubgears
+gdpr_supported: true
```

Comment:
If you claim `gdpr_support` you should add

- gvl_id: none

Reply:
@muuki88 added

Reply:
Do you folks have a `gvl_id`. If not, then please set `gdpr_support` to false